databases/week1/Home work lesson 1.sql

```diff
@@ -0,0 +1,57 @@
+
+-- 1.Find out how many tasks are in the task table
+SELECT count(id) AS TotalTasks
+FROM task;
+
+-- 2.Find out how many tasks in the task table do not have a valid due date
+SELECT count(id) AS TotalTasks
+FROM task
+WHERE task.due_date is NULL;
+
+-- 3.Find all the tasks that are marked as done
+SELECT task.id, task.title, status.name As status
```

Comment:
Avoid shadowing the `status` name. Instead you could call it statusName, which would avoid confusion with the table name status.